packages/cli/cmd/application/main.go

```diff
@@ -77,15 +101,20 @@ func BuildCommandDocsForHugo(cmd *cobra.Command, dir string) error {
 
 func buildRootCmd() *cobra.Command {
 	vars := mainVars{}
+	formatVars := formatVars{}
 	cmd := &cobra.Command{
 		Use:   "agc",
 		Short: shortDescription,
 		Example: `
   Displays the help menu for the specified sub-command.
   /code $ agc account --help`,
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
-			format.SetFormatter(format.FormatterType(vars.format))
 			setLoggingLevel()
+			opts, err := newFormatOpts(formatVars)
+			if err != nil {
+				log.Error().Err(err)
+			}
+			setFormatter(opts)
```

Comment:
Did you notice what happens when "newFormatOpts" fails? in this case setFormatter will be called with empty opts. `storage.ConfigClient` is an interface which is expected to be populated with a pointer to an actual client implementation. In this case it will be `nil` and any attempt to deference this pointer will cause a crash. It would be nice to cover this in unit tests.